### ISSUE TITLE
Dependency Warning

### ISSUE THREAD

**Issue by xandao6** [rpotter12/whatsapp-play#94]
![image](https://user-images.githubusercontent.com/22106326/72111463-a396f600-3319-11ea-8889-22aa75b801b4.png)
Should we care?

**Comment by rpotter12** [comment]
We can think about it later on :)

**Comment by xandao6** [comment]
Solved in Ubuntu 18.04,
```
sudo pip3 install 'urllib3>=1.25.7'
sudo pip3 install --upgrade requests
```
